build.gradle

```diff
@@ -13,11 +13,18 @@ dependencies {
     testImplementation 'org.testng:testng:7.6.1'
     implementation 'org.seleniumhq.selenium:selenium-java:4.5.0'
     testImplementation 'io.github.bonigarcia:webdrivermanager:5.3.1'
+    implementation 'io.cucumber:cucumber-testing:7.9.0'
+    implementation 'io.cucumber:cucumber-core:7.9.0'
+    implementation 'io.cucumber:cucumber-java:7.9.0'
+    implementation 'io.cucumber:gherkin:25.0.2'
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
Your dependencies are inconsistent. You are mixing multiple versions of Cucumber and including transitive dependencies.

![Screenshot_2](https://user-images.githubusercontent.com/104761482/227898734-25ab4133-0940-4f38-b359-2b139c7090aa.jpg)

HINT

```
dependencies {
    testImplementation 'org.testng:testng:7.6.1'
    implementation 'org.seleniumhq.selenium:selenium-java:4.5.0'
    testImplementation 'io.github.bonigarcia:webdrivermanager:5.3.1'
    implementation 'com.opencsv:opencsv:5.7.1'
    implementation 'io.cucumber:cucumber-testng:7.9.0'
    implementation 'io.cucumber:cucumber-core:7.9.0'
    implementation 'io.cucumber:cucumber-java:7.9.0'
    implementation 'io.cucumber:gherkin:25.0.2'


}

```